web/resources/js/hello.ts

```diff
@@ -5,3 +5,14 @@ document.addEventListener('DOMContentLoaded', () => {console.log("This is runnin
 $(function () {
     $('[data-toggle="tooltip"]').tooltip()
 });
+
```

Comment:
Remove this commit from the PR? With `generic-typeahead.ts` there seems to be a proper typescript module now.